# test = 3
number = 0

try:
    print(test / number)
except Exception as e:
    print(type(e))

print("The end")